scripts/spyder_win_post_install.py

```diff
@@ -21,17 +21,95 @@
 KEY_C1 = KEY_C0 + r"\command"
 
 
+# ability to run spyder-win-post-install outside of bdist_wininst installer
+# copied from pywin32-win-post-install

```

Comment:
The [pywin32 mercurial repository is on sourceforge](http://pywin32.hg.sourceforge.net/hgweb/pywin32/pywin32). The most recent build is tagged as [b219](http://pywin32.hg.sourceforge.net/hgweb/pywin32/pywin32/file/b219). Here's a link to  [`pywin32_postinstall.py`](http://pywin32.hg.sourceforge.net/hgweb/pywin32/pywin32/file/b219/pywin32_postinstall.py).